### PR TITLE
feat: Added option to use quotes in sanity editor

### PR DESCRIPTION
**Pull request by KenAJoh**
### Description

https://nav-it.slack.com/archives/C05F2AURXLP/p1716964335567399?thread_ts=1716963408.664719&cid=C05F2AURXLP

### CODE REVIEW

File: aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx [hunk 1]
```diff
@@ -65,10 +65,19 @@ export const block = {
...
+          </span>
+        ),
+        component: ({ children }) => <q>{children}</q>,
+      },
+      {
+        title: "Keyboard",
         value: "kbd",
-        icon: () => <KBD>KBD</KBD>,
-
+        icon: () => <kbd>KBD</kbd>,
```
Comment (HalvorHaugan) [inline aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx:80]: Was this change intended?
image -> image

File: aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx [hunk 2]
```diff
@@ -65,10 +65,19 @@ export const block = {
         component: ({ children }) => <InlineCode>{children}</InlineCode>,
       },
       {
-        title: "KBD",
+        title: "Quote",
+        value: "quote",
+        icon: () => (
+          <span className="font-semibold" aria-label="quote">
```
Comment (HalvorHaugan) [inline aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx]: Looks like Sanity adds `aria-label="Quote"` to the button.
```suggestion
          <span className="font-semibold">
```

File: aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx (hunk 1 above)
Comment (KenAJoh) [reply aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx:80]: Yes, think its a little more in line with how the rest of the richtext editor shows `marks`

File: aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx (hunk 2 above)
Comment (KenAJoh) [reply aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx]: Nice, this was a problem before where none of the buttons had any labels or titles, leading to them being inaccessible for screenreaders.

**Review by HalvorHaugan — APPROVED** [review]
